codice/ddf
# DDF-2190 Updates to the registry local node edit page in the admin ui.

#### What does this PR do?
Updates the registry local nodes edit tab in the admin ui.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@brendan-hofmann  @mcalcote 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@figliold 
@pklinef

#### How should this be tested?
Build and install normally. Install registry and then play around with the local nodes tab in the admin ui to verify functionality of editing/updating/saving of various kinds of fields/configurations
#### Any background context you want to provide?
#### What are the relevant tickets?
DDF-2190
#### Screenshots (if appropriate)
#### Checklist:

  - Added the concept of 'advanced sections/fields' to simplify the ui
  - Moved all fields to the configuration file for fields instead of just having the slots there.
  - Added 'bounds' type that lets the user specify an upper and lower geo corners
  - Fixed some eventing that was a carry over from previouse pr in this area
  - Several other small bug fixes/improvments
